compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1308,16 +1308,33 @@ static SmallVector<int64_t> getConvWorkgroupSizes(func::FuncOp entryPointFn,
   assert(succeeded(variantOp) && "ExecutableVariantOp not found");
 
   if (isX86(*variantOp) || isRISCV(*variantOp)) {
-    tileSizes = {1, 1, 8, vectorSize * 2, 1, 1, 8};
+    TypeSwitch<Operation *>(op.getOperation())
+        .Case<linalg::Conv2DNhwcHwcfOp>(
+            [&](auto op) { tileSizes = {1, 1, 8, vectorSize * 2, 1, 1, 8}; })
+        .Case<linalg::DepthwiseConv2DNhwcHwcOp>(
+            [&](auto op) { tileSizes = {1, 1, 8, vectorSize, 1, 3}; })
```

Comment:
This is also new, right? Should this lead to any performance change?

Reply:
good catch. I thought it's a refactor, but it turns out that the config is changed. It was `{1, 1, 8, vectorSize * 2, 1, 3}`. Let me revert the change.

Reply:
The new values improve performance on RISC-V so I guess we can keep it :)

Reply:
I missed the comment. let's keep the new values for RISC-V side. I'll update the PR title and the code!